examples/iaas_and_iaasalpha/resourse_example.py

```diff
@@ -0,0 +1,187 @@
+        description="This is created via Python SDK",
+        labels={"key": "value"},
+    ),
+)
+security_group1_groupid = security_group1.id
+
+gateway_a_network = client.create_network(
+    project_id,
+    CreateNetworkPayload(
+        name="gateway_a_network",
+    ),
+)
+
+gateway_b_network = client.create_network(
+    project_id,
+    CreateNetworkPayload(
+        name="gateway_b_network",
+    ),
+)
+
+gateway_a_nic1 = alpha_client.create_nic(
+    project_id,
+    gateway_a_network.network_id,
+    CreateNICPayload(
+        allowed_addresses=[x for x in admin_vps_routes],
+        security_groups=[security_group1_groupid],
+    ),
+)
+
+gateway_a_public_ip = alpha_client.create_public_ip(
+    project_id, CreatePublicIPPayload(network_interface=gateway_a_nic1.id)
+)
+
+gateway_b_nic1 = alpha_client.create_nic(
+    project_id,
+    gateway_b_network.network_id,
+    CreateNICPayload(
+        allowed_addresses=[x for x in admin_vps_routes],
+        security_groups=[security_group1_groupid],
+    ),
+)
+
+gateway_b_public_ip = alpha_client.create_public_ip(
+    project_id, CreatePublicIPPayload(network_interface=gateway_b_nic1.id)
+)
+
+gateway_a = alpha_client.create_server(
+    project_id,
+    CreateServerPayload(
+        name="gateway-a",
+        boot_volume=BootVolume(
+            size=64,
+            source=BootVolumeSource(
+                id="db10714b-5d17-4a88-83ea-cb499530acc8",
+                type="image",
+            ),
+        ),
+        keypair_name="key",
+        labels={"key": "value"},
+        machine_type="t1.2",
+        availability_zone="eu01-1",
+    ),
+)
+time.sleep(180)  # wait for server to be created
+alpha_client.add_nicto_server(project_id, gateway_a.id, gateway_a_nic1.id)
+
+
+admin_area_routes = client.create_network_area_route(
+    organization_id,
+    demo_network_area_area_id,
+    CreateNetworkAreaRoutePayload(ipv4=[Route(prefix=x, nexthop=gateway_a_nic1.ipv4) for x in admin_vps_routes]),
+)
+
+
+# Tenant Part (right side of the picture)
+
+tenant_network = client.create_network(
+    project_id,
+    CreateNetworkPayload(
+        name="tenant_network",
+    ),
+)
+
+tenant_nic = alpha_client.create_nic(
+    project_id,
+    tenant_network.network_id,
+    CreateNICPayload(
+        security_groups=[security_group1_groupid],
+    ),
+)
+
+tenant_public_ip = alpha_client.create_public_ip(project_id, CreatePublicIPPayload(network_interface=tenant_nic.id))
+
+
+tenant = alpha_client.create_server(
+    project_id,
+    CreateServerPayload(
+        name="tenant",
+        boot_volume=BootVolume(
+            size=64,
+            source=BootVolumeSource(
+                id="db10714b-5d17-4a88-83ea-cb499530acc8",
+                type="image",
+            ),
+        ),
+        keypair_name="key",
+        labels={"key": "value"},
+        machine_type="t1.2",
+        networking=CreateServerPayloadNetworking(CreateServerNetworkingWithNics(nic_ids=[tenant_nic.id])),
```

Comment:
remove this field, nic is attached below in the method `add_nicto_server`, doesn't need to be provided on creation